package.json

```diff
@@ -43,6 +43,7 @@
     "jest-environment-jsdom": "26.6.1",
     "jwt-decode": "2.2.0",
     "lodash": "4.17.20",
+    "lodash.debounce": "^4.0.8",
```

Comment:
what does this do in the app?

Reply:
It's required by `frontend-enterprise` 😄